server/go/helpers.go

```diff
@@ -10,19 +10,18 @@
 package openapi
 
 import (
+	"errors"
 	"net/http"
 	"reflect"
 
-	kusk "github.com/GIT_USER_ID/GIT_REPO_ID/kusk"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
 )
 
-func handleGetError(err error) (ImplResponse, error) {
-	switch err {
-	case kusk.ErrNotFound:
-		return Response(http.StatusNotFound, err), err
-	default:
-		return Response(http.StatusInternalServerError, err), err
+func GetResponseFromK8sError(err error) ImplResponse {
+	if status := apierrors.APIStatus(nil); errors.As(err, &status) {
+		return Response(int(status.Status().Code), status.Status().Message)
```

Comment:
should we sanitize this somehow so we don't expose potentially sensitive cluster information in error messages!?

Reply:
@jasmingacic `status := apierrors.APIStatus(nil)` what does this line do? I cant see anything in the documentation

Reply:
@kylehodgetts  It creates an instance of APIStatus that i need to cast error into that type 
https://github.com/kubernetes/apimachinery/blob/master/pkg/api/errors/errors.go#L39-L43

@olensmar  There isn't much data coming from there essentially I grab the error code and the message. The same messages you'd get from kubectl 
```
apis.gateway.kusk.io "ttt" not found
```